tools/bug-report/pkg/bugreport/bugreport.go

```diff
@@ -407,19 +406,18 @@ func getLog(client kube.ExtendedClient, resources *cluster2.Resources, config *c
 	return clog, cstat, cstat.Importance(), nil
 }
 
-func runAnalyze(config *config.BugReportConfig, resources *cluster2.Resources, params *content.Params) {
-	for ns := range resources.Root {
-		if analyzer_util.IsSystemNamespace(resource.Namespace(ns)) {
-			continue
-		}
-		common.LogAndPrintf("Running istio analyze on namespace %s.\n", ns)
-		out, err := content.GetAnalyze(params.SetIstioNamespace(config.IstioNamespace))
-		if err != nil {
-			log.Error(err.Error())
-			continue
-		}
-		writeFiles(archive.AnalyzePath(tempDir, ns), out)
+func runAnalyze(config *config.BugReportConfig, params *content.Params) {
+	newParam := params.SetNamespace(common.NamespaceAll)
+	common.LogAndPrintf("Running istio analyze on all namespaces and report as below:")
+	out, err := content.GetAnalyze(newParam.SetIstioNamespace(config.IstioNamespace))
+	if err != nil {
+		log.Error(err.Error())
+		return
 	}
+	common.LogAndPrintf("\n################################################################\n")
```

Comment:
This pattern of printing a bunch of `###` does not seem used elsewhere, do we need it?

Reply:
@howardjohn no, I just want to split the report with some description information.

Reply:
okay, I will remove them.